### ISSUE TITLE
Create threads automatically only after the first "reply"💡 

### ISSUE THREAD

**Issue by dgcohen23** [MarcusOtter/discord-needle#450]
### Describe the improvement

Right now needle creates threads for every post in a channel. It would be great if instead of that, you could have it only do so for posts that have a reply, automatically converting that reply into a thread (and deleting the original reply). This would help keep a server so much more clean!  Basically it would help teach people to reply in threads, instead of with the "reply" function. 

### Problems this improvement solves

Replies are very hard to follow on active Discord servers, particularly if they are replies to older posts. Needle could help alot with this problem by turning the first (or Nth) reply into a thread automatically and deleting the reply. the reply could be posted in the thread and while it might come from the bot, it could tag the original poster of the "reply" and say something like "<their reply> was posted by @thatuser and converted into this thread automatically by needle.gg, since threads are better than replies!"

**Comment by MarcusOtter** [comment]
Duplicate of #93
